ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -603,7 +606,7 @@ def _parse_data_model(self):
                         columns=columns,
                         upstream=upstream_nodes,
                     )
-                    model_fqdn = f"{schema}.{model_name}".lower()
+                    model_fqdn = f"{database}{FQDN_SEPARATOR}{schema}{FQDN_SEPARATOR}{model_name}".lower()
```

Comment:
@ulixius9 why are we not using a method to construct this FQN?

Reply:
There are several places we are not using FQN in sql_source.py. Lets fix these.
@pmbrull lets see if we can merge this https://github.com/open-metadata/OpenMetadata/pull/4134 and use antlr backend and UI is already on it

Reply:
https://github.com/open-metadata/OpenMetadata/issues/4496